src/rgw/rgw_admin.cc

```diff
@@ -1373,7 +1373,7 @@ int check_min_obj_stripe_size(rgw::sal::RGWRadosStore *store, RGWBucketInfo& buc
     auto biter = bl.cbegin();
     decode(manifest, biter);
   } catch (buffer::error& err) {
-    ldpp_dout(store->ctx(), 0) << "ERROR: failed to decode manifest" << dendl;
+    ldout(store->ctx(), 0) << "ERROR: failed to decode manifest" << dendl;
```

Comment:
all of these in `rgw_admin.cc` should be using the `dpp()` declared globablly near the top of the file. So these should be `ldpp_dout(dpp(), 0)`

Reply:
Okay